backend/src/services/auth/auth.service.ts

```diff
@@ -0,0 +1,17 @@
+import { IUser } from '~/common/interfaces/user';
+import { IRegisterPayload } from '~/common/interfaces'
+import { userRepository } from '~/data/repositories';
+
+class AuthService {
+  public async signUp({ registerPayload }: { registerPayload: IRegisterPayload }): Promise<IUser> {
+    const user: IUser = {
+      ...registerPayload,
+      createdAt: new Date(),
+      updatedAt: new Date()
```

Comment:
```suggestion
```
sequlize create it automatically. Remove it

You need to code the password using this package - https://www.npmjs.com/package/bcrypt